azurerm/internal/acceptance/data.go

```diff
@@ -0,0 +1,71 @@
+package acceptance
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/terraform-providers/terraform-provider-azuread/azuread/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+)
+
+type TestData struct {
+	// Location is a set of Azure Regions which should be used for this Test
+	Location Regions
+
+	// RandomString is a random integer which is unique to this test case
+	RandomInteger int
+
+	// RandomString is a random 5 character string is unique to this test case
+	RandomString string
```

Comment:
Where is this used? i've been replacing it where possible by doing a % on the rint

Reply:
Storage/anything Windows related (Redis, VM's etc) - using a combination of letters/numbers gives us a higher chance of uniqueness when we're only allowed 5 characters